Source/IGListKit/IGListCollectionContext+Refinements.swift

```diff
@@ -0,0 +1,69 @@
+//
+// Copyright (c) Facebook, Inc. and its affiliates.
+// All rights reserved.
+//
+// This source code is licensed under the BSD-style license found in the
+// LICENSE file in the root directory of this source tree. An additional grant
+// of patent rights can be found in the PATENTS file in the same directory.
+//
+// GitHub:
+// https://github.com/Instagram/IGListKit
+//
+// Documentation:
+// https://instagram.github.io/IGListKit/
+//
+
+import UIKit
+
+extension ListCollectionContext {
+    /**
+     Dequeues a cell from the collection view reuse pool.
+
+     - Parameters:
+        - cellClass: The class of the cell you want to dequeue.
+        - reuseIdentifier: A reuse identifier for the specified cell. This parameter may be `nil`.
+        - sectionController: The section controller requesting this information.
+        - index: The index of the cell.
+
+     - Returns: A cell dequeued from the reuse pool or a newly created one.
+
+     - Note: This method uses a string representation of the cell class as the identifier.
+     */
+    public func dequeueReusableCell<T: UICollectionViewCell>(
+        of cellClass: T.Type, withReuseIdentifier reuseIdentifier: String,
+        for sectionController: ListSectionController, at index: Int) -> T {
```

Comment:
My nit feedback is to use new line for each individual arguments.

Reply:
Do you feel strongly about this? I grouped them, since the the cell class and reuse identifier are related, and so are the section controller and the index.

Reply:
I dont feel too strongly abt it, which I said it's just a 'nit feedback', up to u  :))